### PR TITLE
[JBPM-9880] logging async node enter

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-9880
depends On 
https://github.com/kiegroup/jbpm/pull/2009
https://github.com/kiegroup/droolsjbpm-knowledge/pull/547

### CODE REVIEW

**Comment by elguardian** [comment]
Jenkins retest this please

File: drools-core/src/main/java/org/drools/core/event/ProcessAsyncNodeScheduledEventImpl.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import java.util.Map;
+
+import org.kie.api.event.process.ProcessAsyncNodeScheduledEvent;
+import org.kie.api.runtime.KieRuntime;
+import org.kie.api.runtime.process.NodeInstance;
+
+public class ProcessAsyncNodeScheduledEventImpl extends ProcessEvent implements ProcessAsyncNodeScheduledEvent {
+
+    private static final long serialVersionUID = 510l;
```
Comment (afalhambra) [inline drools-core/src/main/java/org/drools/core/event/ProcessAsyncNodeScheduledEventImpl.java]: upper L
```suggestion
    private static final long serialVersionUID = 510L;
```

**Review by afalhambra — APPROVED** [review]
Just a minor comment, amazing work

File: drools-core/src/main/java/org/drools/core/event/ProcessAsyncNodeScheduledEventImpl.java (hunk 1 above)
Comment (elguardian) [reply drools-core/src/main/java/org/drools/core/event/ProcessAsyncNodeScheduledEventImpl.java]: done

**Review by afalhambra — APPROVED** [review]
great work!

**Comment by elguardian** [comment]
Jenkins retest this please

**Pull request closed without merging**